src/modules/peek/Peek.Common/Helpers/ReadableStringHelper.cs

```diff
@@ -45,5 +45,13 @@ public static string FormatResourceString(string resourceId, object? args)
 
             return formattedString;
         }
+
+        public static string FormatResourceString(string resourceId, object? args0, object? args1)
+        {
+            var formatString = ResourceLoader.GetForViewIndependentUse().GetString(resourceId);
```

Comment:
Interesting that you found GetForViewIndependentUse() does work here! Just in case this ever does return null, would it be worth doing a null check before calling GetString?